code/controllers/subsystem/time_track.dm

```diff
@@ -59,23 +70,10 @@ SUBSYSTEM_DEF(time_track)
 			"tidi_slowavg",
 			"maptick",
 			"num_timers",
-			"air_turf_cost",
-			"air_eg_cost",
-			"air_highpressure_cost",
-			"air_hotspots_cost",
-			"air_superconductivity_cost",
-			"air_pipenets_cost",
-			"air_rebuilds_cost",
-			"air_turf_count",
-			"air_eg_count",
-			"air_hotspot_count",
-			"air_network_count",
-			"air_delta_count",
-			"air_superconductive_count",
 			"all_queries",
 			"queries_active",
 			"queries_standby"
-		) + sendmaps_headers
```

Comment:
and why are you removing the sendmaps logging?

Reply:
To keep the csv file from killing Google sheets, the lag isn't related to sendmaps, plus this is just to gather other metrics